net/instaweb/rewriter/responsive_image_filter.cc

```diff
@@ -432,9 +432,17 @@ void ResponsiveImageSecondFilter::Cleanup(
 
 void ResponsiveImageSecondFilter::EndDocument() {
   if (zoom_filter_enabled_ && srcsets_added_ && !driver()->is_amp_document()) {
-    HtmlElement* script = driver()->NewElement(NULL, HtmlName::kScript);
-    driver()->AddAttribute(script, HtmlName::kSrc, responsive_js_url_);
-    InsertNodeAtBodyEnd(script);
+    if (driver()->IsLoadPermittedByCsp(
+          CspDirective::kScriptSrc, responsive_js_url_)) {
+      HtmlElement* script = driver()->NewElement(NULL, HtmlName::kScript);
```

Comment:
Done; doesn't seem like I have an integration test in any of the outstanding CLs, though.